exporting/init_connectors.c

```diff
@@ -138,6 +138,7 @@ static size_t base64_encode(unsigned char *input, size_t input_size, char *outpu
             output[3] = '=';
             //error("Base-64 encode (%06x) -> %c %c %c %c\n", (value>>2)&0xffff, output[0], output[1], output[2], output[3]);
             count += 4;
+            output[5] = '\0';
```

Comment:
While we use an output array that is initialized with 0, you are right, it's safer to terminate strings. But don't skip the fourth character.
```suggestion
            output[4] = '\0';
```

Reply:
Yes, U r right.